### PR TITLE
Add to gateway documentation to clarify use of non-standard ports.

### PR DESCRIPTION
**Pull request by jhuttley-F5-com**

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
Hey, the .pb.go files are autogenerated. The .proto is the source of truth. Can you edit that and run `make gen`?

File: networking/v1alpha3/gateway.pb.go [hunk 1]
```diff
@@ -36,6 +36,13 @@
 // my-gateway-controller`. While Istio will configure the proxy to listen
 // on these ports, it is the responsibility of the user to ensure that
 // external traffic to these ports are allowed into the mesh.
+// Ports 9443 and 2379 must exist on the ingress-controller. By default only ports 80 and 443 (http) are configured.
```
[the remark this answers is not in the document]
Comment (jhuttley-F5-com) [reply networking/v1alpha3/gateway.pb.go:39]: Thanks, I've been working backward. Started with the index.html file ...!

[the remark this answers is not in the document]
Comment (jhuttley-F5-com) [reply networking/v1alpha3/gateway.pb.go:39]: ingress-controller, the deployment or  the pods that the gateway configures?

[the remark this answers is not in the document]
Comment (jhuttley-F5-com) [reply networking/v1alpha3/gateway.pb.go:39]: perhaps call it ingress-gateway, let me see what the default is..

[the remark this answers is not in the document]
Comment (jhuttley-F5-com) [reply networking/v1alpha3/gateway.pb.go:39]: yeah the doc is a bit sloppy on separating "gateway" the pods and "gateway" the resource.
helm install istio-ingressgateway istio/gateway -n istio-ingress
I'll see if I can make it clear..

[the remark this answers is not in the document]
Comment (jhuttley-F5-com) [reply networking/v1alpha3/gateway.pb.go:39]: // Ports 9443 and 2379 must exist on the ingress-gateway, `my-gateway-controller` in the example below.
// By default only ports 80 and 443 (http) are configured.
// The available ports are shown in the Service for the ingress-gateway, in the istio-system namespace.
// If a port requested by a Gateway resource does not exist on the ingress-gateway,

[the remark this answers is not in the document]
Comment (jhuttley-F5-com) [reply networking/v1alpha3/gateway.pb.go]: thanks, I'll delete this and redo the PR

**Comment by jhuttley-F5-com** [comment]
will replace

**Pull request closed without merging**